Content.Shared/MouseRotator/SharedMouseRotatorSystem.cs

```diff
@@ -62,10 +66,14 @@ private void OnRequestRotation(RequestMouseRotatorRotationEvent msg, EntitySessi
 
     private void OnRequestSimpleRotation(RequestMouseRotatorRotationSimpleEvent ev, EntitySessionEventArgs args)
     {
-        if (args.SenderSession.AttachedEntity is not { } ent
-            || !TryComp<MouseRotatorComponent>(ent, out var rotator) || !rotator.Simple4DirMode)
+        if (args.SenderSession.AttachedEntity is not { } ent)
+            return;
+
+        var rotator = EnsureComp<MouseRotatorComponent>(ent);
```

Comment:
so the main issue with ensuring it is that now literally any time a client sends the rotation request message, it's going to add the rotator component to them regardless of whether or not they're supposed to have it.

I think the actual issue is just some network-lag chicanery that results in the component not always being there when it needs to. You could probably just remove the error logging considering it's not actually a major issue?

Reply:
Pretty sure it causes desync with client's stuff being dropped.

Reply:
> Pretty sure it causes desync with client's stuff being dropped.

Are you agreeing with emo that I should just go back to how it was before with the error removed or are you saying something else...?